pyrasterframes/src/main/python/setup.py

```diff
@@ -145,6 +145,7 @@ def initialize_options(self):
         'ipython==6.2.1',
         "ipykernel==4.8.0",
         'Pweave==0.30.3',
+        'fiona==1.8.6'
```

Comment:
What was the reason for this? I thought fiona would come in with Shapely? see line 136?

But having said that I also had problems with shapely a few commits later.

Reply:
I had an issue building the site because I had an outdated version of fiona, so Sim added this to prevent other people from having the same issue.